Framework/Geometry/inc/MantidGeometry/MDGeometry/MDHistoDimension.h

```diff
@@ -113,6 +113,12 @@ class MANTID_GEOMETRY_DLL MDHistoDimension : public IMDDimension {
     m_binWidth = (m_max - m_min) / static_cast<coord_t>(m_numBins);
   }
 
+  /**
+   * Set the MDFrame
```

Comment:
I think we should be adding some text to this later to warn against using this. Setting the frame on an existing MDHistoDimension should not be a common use-case.